repo_id,repo_full_name,file,file_sha1,match
269130305,ynots0ups/CVE-2019-16113,cve-2019-16113.py,70999f40aebff8d959a69506c82c95bfb4bc7722,CVE-2019-16113
269130305,ynots0ups/CVE-2019-16113,README.md,7aa05de5e1b9a011554057d20fa01c6403a94314,CVE-2019-16113
269130305,ynots0ups/CVE-2019-16113,_GITHUB_REPO_METADATA_,,CVE-2019-16113
